src/components/mapPageComponents/mapLegend/index.tsx

```diff
@@ -0,0 +1,191 @@
+                  <>
+                    <Text strong>Trees represented by</Text>
+                    <br />
+                    <Row>
+                      <CenterCol span={3}>
+                        <LegendIcon src={youngTreeIcon} preview={false} />
+                      </CenterCol>
+                      <Col span={21}>
+                        <FlexibleParagraph fontSize={fontSize}>
+                          {YOUNG_TREE_ICON_DESCRIPTION}
+                        </FlexibleParagraph>
+                      </Col>
+                    </Row>
+                    <Row>
+                      <CenterCol span={3}>
+                        <LegendIcon src={treeIcon} preview={false} />
+                      </CenterCol>
+                      <Col span={21}>
+                        <FlexibleParagraph fontSize={fontSize}>
+                          {TREE_ICON_DESCRIPTION}
+                        </FlexibleParagraph>
+                      </Col>
+                    </Row>
+                    <Row>
+                      <CenterCol span={3}>
+                        <LegendIcon src={adoptedTreeIcon} preview={false} />
+                      </CenterCol>
+                      <Col span={21}>
+                        <FlexibleParagraph fontSize={fontSize}>
+                          {ADOPTED_TREE_ICON_DESCRIPTION}
+                        </FlexibleParagraph>
+                      </Col>
+                    </Row>
+                  </>
+                )}
+              </>
+            );
+          case MapViews.BLOCKS:
+            return (
+              <>
+                {showLegend && (
+                  <>
+                    <Text strong>Blocks that are colored</Text>
+                    <br />
+                    <Row>
+                      <CenterCol span={3}>
+                        <ColorBlock color={MAP_GREEN} />
+                      </CenterCol>
+                      <Col span={21}>
+                        <FlexibleParagraph fontSize={fontSize}>
+                          {OPEN_BLOCK_DESCRIPTION}
+                        </FlexibleParagraph>
+                      </Col>
+                    </Row>
+                    <Row>
+                      <CenterCol span={3}>
+                        <ColorBlock color={MAP_YELLOW} />
+                      </CenterCol>
+                      <Col span={21}>
+                        <FlexibleParagraph fontSize={fontSize}>
+                          {RESERVED_BLOCK_DESCRIPTION}
+                        </FlexibleParagraph>
+                      </Col>
+                    </Row>
+                    <Row>
+                      <CenterCol span={3}>
+                        <ColorBlock color={MAP_RED} />
+                      </CenterCol>
+                      <Col span={21}>
+                        <FlexibleParagraph fontSize={fontSize}>
+                          {CLOSED_BLOCK_DESCRIPTION}
+                        </FlexibleParagraph>
```

Comment:
📌 Not necessary for this ticket, but we don't currently have a color on the map for completed blocks. Once that is implemented, we'll need to add to the legend

Reply:
Updated [this ticket](https://code4community-team.monday.com/boards/925293934/views/18904377/pulses/1114569938) for that